<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>minimumAreaTool</class>
 <widget class="QWidget" name="minimumAreaTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>45</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>474</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QComboBox" name="scalesComboBox">
      <item>
       <property name="text">
        <string>SCALES</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>100</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="sizesComboBox">
      <item>
       <property name="text">
        <string>SIZES</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4mm²</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0.8x0.8mm²</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25mm²</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1x1mm²</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0.8mm</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="shapesComboBox">
      <item>
       <property name="text">
        <string>SHAPES</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Square</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hexagon</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="drawShape">
      <property name="text">
       <string>Draw Shape</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/plugins/DsgTools/icons/areaTool.png</normaloff>:/plugins/DsgTools/icons/areaTool.png</iconset>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
